cat-launcher/src-tauri/src/mods/get_mod_activity.rs

```diff
@@ -0,0 +1,99 @@
+use std::collections::HashMap;
+use std::io;
+use std::path::Path;
+use serde::Deserialize;
+use tokio::fs::read_to_string;
+use reqwest::Client;
+use crate::mods::paths::get_mods_resource_path;
+use crate::mods::types::{ThirdPartyMod};
+use crate::variants::GameVariant;
+
+
+#[derive(Debug, Deserialize)]
```

Comment:
The GitHub commit, commit details, and signature types should be moved into the existing infra/github module.